src/main/java/com/denizenscript/depenizen/bukkit/objects/mythicmobs/MythicMobsMobTag.java

```diff
@@ -366,6 +399,25 @@ public void applyProperty(Mechanism mechanism) {
     @Override
     public void adjust(Mechanism mechanism) {
 
+        // <--[mechanism]
+        // @object MythicMobsMobTag
+        // @name mythic_variable_map
+        // @input MapTag
+        // @plugin Depenizen, MythicMobs
+        // @description
+        // Sets the variables of a MythicMob from a map.
+        // @tags
+        // <MythicMobsMobTag.mythic_variable_map>
+        // -->
+        if (mechanism.matches("mythic_variable_map") && mechanism.requireObject(MapTag.class)) {
+            MapTag map = mechanism.valueAsType(MapTag.class);
+            Map<String, Variable> new_map = new HashMap<String, Variable>();
```

Comment:
Should be `newMap` or `variableMap` or whatever for naming consistency with the rest of the codebase